receiver/windowsperfcountersreceiver/internal/pdh/pdh_test.go

```diff
@@ -80,12 +80,50 @@ func TestPerfCounter_Close(t *testing.T) {
 }
 
 func TestPerfCounter_ScrapeData(t *testing.T) {
-	pc, err := NewPerfCounter(`\Memory\Committed Bytes`, false)
-	require.NoError(t, err)
+	type testCase struct {
+		name           string
+		path           string
+		assertExpected func(t *testing.T, data []win_perf_counters.CounterValue)
+	}
 
-	performanceCounters, err := pc.ScrapeData()
-	require.NoError(t, err, "Failed to scrape data: %v", err)
+	testCases := []testCase{
+		{
+			name: "no instances",
+			path: `\Memory\Committed Bytes`,
+			assertExpected: func(t *testing.T, data []win_perf_counters.CounterValue) {
+				assert.Equal(t, 1, len(data))
+				assert.Empty(t, data[0].InstanceName)
+			},
+		},
+		{
+			name: "total instance",
+			path: `\LogicalDisk(_Total)\Free Megabytes`,
+			assertExpected: func(t *testing.T, data []win_perf_counters.CounterValue) {
+				assert.Equal(t, 1, len(data))
+				assert.Empty(t, data[0].InstanceName)
+			},
+		},
+		{
+			name: "all instances",
+			path: `\LogicalDisk(*)\Free Megabytes`,
```

Comment:
Is the instance name here "*" (literal)?

Reply:
`*` is a wildcard that means all instances

There's three types of values you can specify here:

- `instance_name`: returns results for the specified instance.
- `_Total`: special value that returns results for the total across all instances.
- `*`: special value that returns separate results for all instances (including totals).

This is described in the README [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/receiver/windowsperfcountersreceiver/README.md#configuration).